recipes/kombu/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "3.0.26" %}
+
+package:
+  name: kombu
+  version: {{ version }}
+
+source:
+  fn: kombu-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/k/kombu/kombu-{{ version }}.tar.gz
+  md5: dda05763cd4b72a131fe1ef9ba93d542
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - anyjson >=0.3.3
+    - amqp >=1.4.6,<2.0
+
+  run:
+    - python
+    - anyjson >=0.3.3
+    - amqp >=1.4.6,<2.0
+    - importlib  # [py26]
+    - ordereddict  # [py26]
+
+test:
+  imports:
+    - kombu
+    - kombu.async
+    - kombu.tests
+    - kombu.tests.async
+    - kombu.tests.transport
+    - kombu.tests.transport.virtual
+    - kombu.tests.utils
+    - kombu.transport
+    #- kombu.transport.django
+    #- kombu.transport.django.management
+    #- kombu.transport.django.management.commands
+    #- kombu.transport.django.migrations
+    #- kombu.transport.sqlalchemy
```

Comment:
Erm ... I forget.
I'm guessing they're meant to be optional public entrypoints, but fail to import without the optional dependencies django and sqlalchemy installed.  See also `oslo.messaging/meta.yaml` where I had a similar problem.

In general, I wasn't sure what I should do with optional dependencies.  I can require them just for testing (which then means they need to be packaged, but I see django already is), or I can ignore them completely and not test them.  Is there a suggested approach for this?